README.md

```diff
@@ -1,17 +1,47 @@
-# Bitcoin Curriculum
+# What is this?
+
+While planning the [Chaincode Residency](https://residency.chaincode.com/), we put considerable effort into finding the best resources and creating a curriculum around Bitcoin protocol development.
+
+Bitcoin is constantly evolving, and so we expect maintenance of this document to be an ongoing task. We could use your help. Please **consider opening a pull-request** to keep this document relevant.
+
+There are three portions to this curriculum:
+1. [Study Groups](https://github.com/chaincodelabs/study-groups#bitcoin) designed to provide bite-sized grouped subjects that you can either complete bookclub style or alone.
```

Comment:
Yeah, I'd rather not commit to a number of hours and it's another repo because I'd like to group all the study groups together even though they span both Bitcoin and Lightning.